File: src/main/java/net/xalcon/technomage/common/CreativeTabTechnomage.java
package net.xalcon.technomage.common;

import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.init.Items;
import net.minecraft.item.ItemStack;

public class CreativeTabTechnomage
{
    public static CreativeTabs Main = new CreativeTabs("technomage")
    {
        @Override
        public ItemStack getTabIconItem()
        {
            return new ItemStack(Items.ENCHANTED_BOOK);
        }
    };
}

File: src/main/kotlin/net/xalcon/technomage/common/CreativeTabTechnomage.kt
package net.xalcon.technomage.common

import net.minecraft.creativetab.CreativeTabs
import net.minecraft.init.Items
import net.minecraft.item.ItemStack
import net.xalcon.technomage.Technomage

object CreativeTabTechnomage : CreativeTabs(Technomage.MOD_ID)
{
    override fun getTabIconItem(): ItemStack
    {
        return ItemStack(Items.BOOK)
    }
}